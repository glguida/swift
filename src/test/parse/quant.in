query exists Person x exists Person y (x==y);