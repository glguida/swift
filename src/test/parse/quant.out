(BlogProgram:
  (args:
    (Query:
      :expr
        (QuantExpr:
          :typ 19
          (VarDecl: (: Person x ) )
          :cond
            (QuantExpr:
              :typ 19
              (VarDecl: (: Person y ) )
              :cond
                (OpExpr:
                  :op 6
                  :left
                    (VarRef: x )
                  :right
                    (VarRef: y )
                )
            )
        )
    )
  )
)
